test/sagemaker_tests/mxnet/training/conftest.py

```diff
@@ -90,7 +91,10 @@ def pytest_generate_tests(metafunc):
 def docker_base_name(request):
     return request.config.getoption('--docker-base-name')
 
-
+@pytest.fixture(scope='session', name='multi_region_support')
+def fixture_multi_region(request):
```

Comment:
Please return `None` for the fixture `n_virginia_sagemaker_session` and `n_virginia_ecr_image` if `not multi_region_support`, as an input guard. This will ensure that regions in other partitions do not fail, looking for access to `us-east-1`.

Reply:
Good catch. Will make the change